config/database.yml

```diff
@@ -19,27 +19,44 @@ default: &default
   # https://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
 
-development:
-  <<: *default
-  database: eths_indexer_development
-
   # The specified database role being used to connect to PostgreSQL.
   # To create additional roles in PostgreSQL see `$ createuser --help`.
   # When left blank, PostgreSQL will use the default role. This is
   # the same name as the operating system user running Rails.
-  #username: eths_indexer
+  username: <%= ENV["DATABASE_USERNAME"] %>
 
   # The password associated with the PostgreSQL role (username).
-  #password:
+  password: <%= ENV["DATABASE_PASSWORD"] %>
 
   # Connect on a TCP socket. Omitted by default since the client uses a
   # domain socket that doesn't need configuration. Windows does not have
   # domain sockets, so uncomment these lines.
-  #host: localhost
+  host: <%= ENV["DATABASE_HOST"] %>
 
   # The TCP port the server listens on. Defaults to 5432.
   # If your server runs on a different port number, change accordingly.
-  #port: 5432
+  port: <%= ENV["DATABASE_PORT"] %>
+
+  # Instead, provide the password or a full connection URL as an environment
+  # variable when you boot the app. For example:
+  #
+  #   ETHS_INDEXER_DATABASE_URL="postgres://myuser:mypass@localhost:port/somedatabase"
+  #
+  # If the connection URL is provided in the special DATABASE_URL environment
+  # variable, Rails will automatically merge its configuration values on top of
+  # the values provided in this file. Alternatively, you can specify a connection
+  # URL environment variable explicitly:
+  #
+  #   production:
+  #     url: <%= ENV["DATABASE_URL"] %>
+  #
+  # Read https://guides.rubyonrails.org/configuring.html#configuring-a-database
+  # for a full overview on how database connection configuration can be specified.
+  url: <%= ENV["DATABASE_URL"] %>
+
+development:
+  <<: *default
+  database: eths_indexer_development
```

Comment:
This should be pulled from ENV, no? Also maybe we call it "DATABASE_BASENAME" or something so it makes sense to append `_development` etc. Thoughts?

Reply:
Your suggestion is great! 😃
I've added the database name configuration to the ENV, so everyone can name the database according to their needs.